aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -1,6 +1,7 @@
 #include <ATen/ATen.h>
 #include <ATen/quantized/Quantizer.h>
 #include <c10/core/Allocator.h>
+#include <c10/core/CPUAllocator.h>
```

Comment:
do we need this import? looks like `at::getCPUAllocator()` works before

Reply:
Thanks, removed.

Reply:
Apparently it's needed.

Reply:
Oh, I see, it's for `c10::alloc_cpu` I think.